build.gradle.kts

```diff
@@ -93,11 +95,11 @@ yamlValidator {
 }
 
 jacoco {
-	toolVersion = "0.8.6"
+	toolVersion = "0.8.7"
 }
 
 sonarqube {
-	// If you want to use this logcally a sonarLogin has to be provide, either via Username and Password
+	// If you want to use this logically a sonarLogin has to be provided, either via Username and Password
```

Comment:
Wait, isn't this supposed to be "locally"? 😅

Reply:
Ah, that makes a lot more sense 😅
Have fixed it now, sorry.